Java/core/src/main/java/com/amazon/randomcutforest/tree/CompactRandomCutTreeDouble.java

```diff
@@ -55,21 +56,23 @@
     public static final short NULL = -1;
 
     private final Random random;
-    private final int maxSize;
+    protected final short maxSize;
```

Comment:
What does this variable do? Can we give it a clearer name?

Reply:
Not used in the most recent refactoring. We can use the old version.